org.videolan.VLC.Plugin.pause_click.appdata.xml

```diff
@@ -0,0 +1,67 @@
+    </p>
+  </description>
+  <categories>
+    <category>Video</category>
+    <category>AudioVideo</category>
+    <category>Player</category>
+  </categories>
+  <content_rating type="oars-1.1" />
+  <url type="homepage">https://github.com/nurupo/vlc-pause-click-plugin</url>
+  <url type="bugtracker">https://github.com/nurupo/vlc-pause-click-plugin/issues</url>
+  <url type="faq">https://github.com/nurupo/vlc-pause-click-plugin/blob/master/README.md</url>
+  <url type="donation">https://github.com/sponsors/nurupo</url>
+  <url type="donation">https://www.paypal.com/donate?hosted_button_id=9HJHAH5UDL3GL</url>
+  <url type="donation">https://blockstream.info/address/34qxFsZjs1ZWVBwer11gXiycpv7QHTA8q3</url>
+  <url type="vcs-browser">https://github.com/nurupo/vlc-pause-click-plugin</url>
+  <releases>
+    <release version="2.2.0" date="2020-05-19"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/2.2.0</url></release>
+    <release version="2.1.0" date="2019-07-30"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/2.1.0</url></release>
+    <release version="2.0.0" date="2019-04-21"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/2.0.0</url></release>
+    <release version="1.0.0" date="2018-02-09"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/1.0.0</url></release>
+    <release version="0.4.0" date="2017-11-22"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.4.0</url></release>
+    <release version="0.3.2" date="2017-06-24"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.3.2</url></release>
+    <release version="0.3.1" date="2017-02-22"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.3.1</url></release>
+    <release version="0.3.0" date="2016-03-06"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.3.0</url></release>
+    <release version="0.2.1" date="2015-09-24"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.2.1</url></release>
+    <release version="0.2.0" date="2014-11-09"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.2.0</url></release>
+    <release version="0.1.0" date="2014-09-30"><url>https://github.com/nurupo/vlc-pause-click-plugin/releases/tag/0.1.0</url></release>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Plugin settings</caption>
+      <image type="source">https://i.imgur.com/Kdrekks.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Checkbox enabling the plugin under Interface menu</caption>
+      <image type="source">https://i.imgur.com/m9yF5Px.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Checkbox enabling the plugin under Video menu</caption>
+      <image type="source">https://i.imgur.com/OZLqmI6.png</image>
+    </screenshot>
+  </screenshots>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
usually for thr metadata, `CC0-1.0` is preferred

Reply:
I would prefer to keep it CC-BY-SA-3.0 if possible.

Could you elaborate more on the issues using CC-BY-SA-3.0 might cause?